### DISCUSSION TITLE
Full thing for Amazon Fire tablet

### DISCUSSION THREAD

**Discussion by Shiro2012** [kinkofer/FightClub5eXML#201] [Q&A]
I'm wondering if you could download it for the fire tablet, I want to use it for my campaigns.

**Comment by abc** [comment]
Hey @Shiro2012, just to let you know, this repo is unfortunately abandoned and development has moved over to ogre-project/FightClub5eXML for the time being, at least until @kinkofer returns and adds some contributors to this repo.

You should be able to side load FightClub5E onto the Kindle Fire without issue. For the XML file, you can download it directly from the releases page of the ogre-project repo (above). 

Any problems or questions let me know!
